capabilities.js

```diff
@@ -4,6 +4,19 @@ define('capabilities', ['settings'], function(settings) {
         return !!m && m.matches;
     }
 
+    function detectOS() {
+        var macInfo = /^[^\(]+\(Macintosh.+Mac OS X 10[_\.](\d+)[^\)]+\)/
```

Comment:
firefox says `Mozilla/5.0 (Macintosh; Intel Mac OS X 10.10; rv:37.0) Gecko/20100101 Firefox/37.0`
chrome says `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36`

notice dots versus underscores in the OS X version numbers. if you want this to work only in Firefox, this is fine. but realise this won't scale if/when we have apps installable elsewhere.